dist/index.d.ts

```diff
@@ -1995,7 +1995,14 @@ interface GPUQueue
   copyExternalImageToTexture(
     source: GPUImageCopyExternalImage,
```

Comment:
hmm, I just noticed the definition of `GPUImageCopyExternalImage` seems to be missing from this file